kedro/extras/datasets/holoviews/holoviews_writer.py

```diff
@@ -21,7 +21,7 @@
 HoloViews = TypeVar("HoloViews")
 
 
-class HoloviewsWriter(AbstractVersionedDataSet):
+class HoloviewsWriter(AbstractVersionedDataSet[HoloViews, str]):
```

Comment:
```suggestion
class HoloviewsWriter(AbstractVersionedDataSet[HoloViews, None]):
```
```suggestion
class JSONDataSet(AbstractVersionedDataSet[pd.DataFrame, pd.DataFrame]):
```
And change the type signature for `_load` 🙂